infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -31,12 +32,20 @@
 
     private final ServerContext serverContext;
 
+
     /**
      * Interval in checking presence of management layout to start management agent tasks.
-     * In milliseconds.
+     * In seconds.
      */
     @Getter
     private static final Duration CHECK_BOOTSTRAP_INTERVAL = Duration.ofSeconds(1);
+
+    /**
+     * Interval in retrying layout recovery when management server started.
+     * In seconds.
+     */
+    private static Duration RECOVERY_RETRY_INTERVAL = Duration.ofSeconds(1);
```

Comment:
Can we make this variable `final`? If not then why this is static?

Reply:
Done.